src/use-swr-infinite.ts

```diff
@@ -118,6 +118,8 @@ function useSWRInfinite<Data = any, Error = any>(
     } else {
       didMountRef.current = true
     }
+    // initialSize isn't allowed to change during the lifecycle
```

Comment:
SWR doesn't support changing `initialSize` during the lifecycle, does it?

Reply:
No it doesn't (it's hard to define what should happen if you change that option), maybe we can have a common util to show warning messages like this and #898?